app/uk/gov/hmrc/vatsubscription/models/VatCustomerInformation.scala

```diff
@@ -118,15 +119,15 @@ object VatCustomerInformation extends JsonReadUtil with JsonObjectSugar {
     flatRateScheme,
     ppob,
     bankDetails,
-    filterReturnPeriod(returnPeriod),
+    filterReturnPeriod(returnPeriod, conf),
     deregistration,
     changeIndicators,
     pendingChanges,
     partyType,
     primaryMainCode
   )
 
-  val release10Reads: Reads[VatCustomerInformation] = for {
+  val release10Reads: AppConfig => Reads[VatCustomerInformation] = conf => for {
```

Comment:
If we're using this in live now it should be R11 reads. Just to make it a bit clearer.

Reply:
Discussed in person and this comment isn't needed anymore.